SoftLayer/CLI/registration/contacts.py

```diff
@@ -0,0 +1,40 @@
+"""Get Contact details for a subnet registration."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """Display the RIR contact information for your account.
+
+    """
+    mgr = SoftLayer.NetworkManager(env.client)
+    result = mgr.get_registration_details()
+    table = formatting.Table(['id', 'name', 'email', 'label', 'address', 'phone'])
+    for contact in result:
+        json_result = {'EMAIL_ADDRESS': '', 'INTERNAL_LABEL': ''}
+        json_result['id'] = contact['id']
+        for property_contact in contact['properties']:
+            if property_contact['propertyType']['keyName'] == 'EMAIL_ADDRESS':
```

Comment:
what is the point of this if/else block? I dont understand why `json_result[property_contact['propertyType']['keyName']] = (property_contact['value'])` doesn't work for EMAIL/INTERNAL_LABEL